src/components/forms/ClientRecordForm/index.tsx

```diff
@@ -104,42 +133,66 @@ const UPDATE_CLIENT = gql`
     }
 `;
 
+const CLIENT_OPTIONS = gql`
+    query ClientOptions {
+        useCaseTypes: __type(name: "USE_CASE_TYPES") {
+            name
+            enumValues {
+                name
+                description
+            }
+        }
+    }
+`;
+
 type ClientRecordFormFields = CreateClientMutationVariables['clientRecordItem'];
 type FormType = PurgeNull<PartialForm<WithId<ClientRecordFormFields>>>;
 
 type FormSchema = ObjectSchema<FormType>;
 type FormSchemaFields = ReturnType<FormSchema['fields']>;
 
 const schema: FormSchema = {
-    fields: (): FormSchemaFields => ({
-        id: [idCondition],
-        name: [requiredCondition],
-        code: [requiredStringCondition],
-        isActive: [],
-    }),
-};
+    fields: (val): FormSchemaFields => {
+        const baseSchema: FormSchemaFields = ({
+            id: [],
+            acronym: [],
+            name: [requiredCondition],
+            contactName: [requiredCondition],
+            contactEmail: [requiredCondition, emailCondition],
+            contactWebsite: [urlCondition],
+            isActive: [],
+            useCases: [],
+            optedOutOfEmails: [requiredCondition],
+        });
 
-const defaultFormValues: PartialForm<FormType> = {
-    code: undefined,
-    name: undefined,
-    isActive: undefined,
+        if (val?.useCases?.includes('OTHER')) {
+            return {
+                ...baseSchema,
+                otherNotes: [requiredCondition],
+            };
+        }
+        return {
+            ...baseSchema,
+            otherNotes: [],
```

Comment:
```suggestion
            otherNotes: [nullCondition],
```

We want to clear the notes when use case does not have "OTHER"